content/spin/serverless-ai-tutorial.md

```diff
@@ -106,24 +106,79 @@ HTTP path: /api/...
 {{ blockEnd }}
 {{ blockEnd }}
 
-### Fetch AI Model
+### Fetching Supported AI Models
 
-Next, we create a folder and fetch a pre-trained AI model for our application:
+Next, **from within the application directory (alongside our `spin.toml` file), we create the appropriate folder structure (depending on which model we want to use) and then fetch the pre-trained AI model for our application:
```

Comment:
I think we need to add a disclaimer up front about this:

> Radu: We should mention that people should agree to Llama2's TOS
@radu-matei Not sure how you think we can accomplish this? And still give away the direct download link. Thoughts?

Reply:
I think the Policy Requirements heading and the text suffice for a disclaimer. Please let me know if more is needed.